core/controllers/profile_test.py

```diff
@@ -864,7 +864,10 @@ def test_user_info_handler(self):
             'user_is_logged_in': True}, json_response)
```

Comment:
I suggest modifying this to check the full dict, not just a subset.

Reply:
Done